Source/WTF/wtf/StdLibExtras.h

```diff
@@ -832,6 +862,18 @@ T& reinterpretCastSpanStartTo(std::span<uint8_t, Extent> span)
     return spanReinterpretCast<T>(span.first(sizeof(T)))[0];
 }
 
+template<typename T, std::size_t Extent>
+const T& reinterpretCastSpanStartTo(std::span<const std::byte, Extent> span)
```

Comment:
This additional overloading is part of the cost of starting to use `std::byte`. I suppose `byteCast` sprinkling is less elegant.